src/app/excel/worker.ts

```diff
@@ -0,0 +1,7 @@
+﻿export class Worker {
```

Comment:
taki interfejs proponuję dodać na końcu pliku w którym jest wykorzystywany
jakoś tak się w tutorialach przyjęło, że robi się interfejsy - wydaje mi się że tutaj też byłby wystarczający (no ale nie znam się jak to dokładnie działa pod spodem)